frontend/lib/requests/user-activity.ts

```
@@ -256,19 +205,66 @@ function formatPostHogDescription(event: any): string {
+  ) {
+    const questionTitle = properties.question_title || "";
+    const truncatedTitle =
+      questionTitle.length > 50
+        ? questionTitle.substring(0, 50) + "..."
+        : questionTitle;
+
+    return `Quiz attempted: "${truncatedTitle}"`;
+  }
+  if (eventName.toLowerCase().includes("lesson_completed")) {
+    const lessonName = properties.lesson_name || properties.lessonName;
+    const dropletId = properties.droplet_id;
```

ESLint

[error] 'dropletId' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)